src/_includes/avatar.njk

```diff
@@ -0,0 +1,14 @@
+{% if avatar.link %}
+  <a
+    class="c-avatar {{ avatar.cssClass }}"
+    style="background-image: url({{ avatar.image }});"
+    href="{{ avatar.link }}"
+  >
+    <div class="u-hide-visually">{{ avatar.name }}'s avatar</div>
+  </a>
```

Comment:
I'd like to remove this and re-insitute the inline `onclick="location.href='{{ avatar.url }}"` JavaScript we're currently using.

The reasoning for this is the person name's is wrapped in an anchor element, and provides an accessible name for assistive technology. 

`{{ avatar.name }}'s avatar` does not provide much in the way of information, and adding it as an anchor introduces a second, redundant link. This can make traversing a list of links more annoying, or even cumbersome when severe motor control issues are present.

It's a little counterintuitive hack that makes for a better assistive technology experience 🙂

Reply:
Gotcha- let me revert this to the past approach 👍 I think the case of the spotlight image being an `img` element with an alt text value vs the other instances being `div`s made me overthink things a bit here.

Appreciate the context for the original approach as well- that balancing act of reducing static while still giving enough context is always tricky, and I'm always interested to learn how others are approaching these kinds of situations.